### DISCUSSION TITLE
About the startup of debug and IProcess

### DISCUSSION THREAD

**Discussion by yangmeng66** [eclipse-platform/eclipse.platform.releng.aggregator#895] [Q&A] [answered]
Hello, 
I'm studying: https://www.vogella.com/tutorials/EclipseLauncherFramework/article.html#how-a-launch-can-be-triggered

I tried to start the demo code, but I don't know how to write this "IProcess" at this stage

**Comment by merks** [comment, marked as answer]
This is really not the place to ask...

I can only suggest that you look at how others call that method to see how they do things, e.g., here the call hierarchy shows a bunch of places you could look:

![image](https://user-images.githubusercontent.com/208716/220903952-e8ea93b6-9e7b-4382-9598-9d7e829e76a8.png)

**Reply by yangmeng66** [reply]
I'm very sorry, thank you for your reply
